decidim-elections/config/locales/en.yml

```
@@ -1291,6 +1297,7 @@ en:
           day_placeholder: DD
           document_number: Document number
           document_number_placeholder: ID number
+          document_type: Tipus de document
```

```suggestion
          document_type: Document type
```

Uops catalan string, this needs to be changed

---

I see that have some failing specs that are related to this, so they'll pass when this is changed https://github.com/decidim/decidim/runs/5705110331?check_suite_focus=true

---

Ouf! Rookie mistake. Thanks!

---

It seems like it's still there, can you check it again?